app/views/homepage/popular_links/show.html.erb

```diff
@@ -7,6 +7,6 @@
     <%= render "homepage/popular_links/link", item:, index: %>
   <% end %>
 </div>
-<div class = "govuk-grid-column-one-third" >
+<div class = "govuk-grid-column-one-third sticky" >
```

Comment:
Hmm, so I think this has sparked a naming convention debate (in my head at least). Probably the SCSS file should be called `popular-links`. Then my suggestion would be to use the name `popular-links__sidebar-components` (or something like it) for this element and maybe rename the child div of this to `sidebar-components__contents` (though strictly speaking it probably doesn't need a name at all). Then duplicate those names in the appropriate places in the Sass.

Reply:
Think I'd go for something like this: 
```
<div class = "govuk-grid-column-one-third popular-links__sidebar">
  <div class="sidebar__components">
```